sos/plugins/sas3ircu.py

```diff
@@ -0,0 +1,47 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+
+# This sosreport plugin is meant for sas adapters.
+# This plugin logs inforamtion on each adapter it finds.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os
+
+
+class SAS3ircu(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+
+    plugin_name = "sas3ircu"
+
+    def setup(self):
+
+        # get list of adapters
+        output = self.call_ext_prog("sh -c 'sas3ircu list'", timeout=5)
+
```

Comment:
Yes, that's exactly what my plugin wants. I'm not sure how common of a problem this is but if you think it would be useful then that works for me. 
I made some changes and added the "commands" line so it should be ready whenever you have "commands" working.